ui/tests/acceptance/allocation-detail-test.js

```diff
@@ -1,8 +1,8 @@
-import $ from 'jquery';
 import { assign } from '@ember/polyfills';
-import { click, findAll, currentURL, find, visit } from 'ember-native-dom-helpers';
+import { currentURL } from 'ember-native-dom-helpers';
 import { test } from 'qunit';
 import moduleForAcceptance from 'nomad-ui/tests/helpers/module-for-acceptance';
+import Allocation from 'nomad-ui/tests/pages/allocations/detail';
```

Comment:
Out of interest, just wondering why you chose to go with 'Allocation' vs 'allocation' (etc etc for other page objects)?

Reply:
I didn't like what the docs do (which is use `page`). I didn't find that particularly helpful to read, and since some tests rely on multiple page objects, the pattern doesn't hold up.

I also didn't want to step on any variables that would likely be used for tests (e.g., `allocation`).

The final thought was I didn't want to give the impression that page objects were some ephemeral thing being used as a means to an end in tests (again like `allocation` or the pattern seen throughout of `const row = Page.collection.objectAt(0)`. So I went with the OOPy capital-case singleton vibe.

Reply:
👍 nice